README.md

```diff
@@ -1 +1,39 @@
-# esb-connector-viber
\ No newline at end of file
+#Viber EI Connector
+
+The Viber [connector](https://docs.wso2.com/display/EI640/Working+with+Connectors) is a free to download app that allows users to make free calls and send messages to other Viber users .It works on both mobile and computer and can
+be used to connect with people around the world.
+It allows you to access the [Viber REST API](https://developers.viber.com/docs/api/rest-bot-api/) through WSO2 Enterprise Integrator (WSO2 EI).
+
+
+## Compatibility
+
+| Connector Version | Viber API Version | Supported WSO2 EI Version |
+| ------------- | ------------- | ------------- |
+| [1.0.0](https://github.com/wso2-extensions/esb-connector-viber/releases/tag/org.wso2.carbon.connector.viber-1.0.0)  | V7.3.0| ESB 6.4.0 |
```

Comment:
I think link should be
https://github.com/wso2-extensions/esb-connector-viber/releases/tag/org.wso2.carbon.esb.connector.viber-1.0.0
Check this and update